include/zephyr/drivers/dma/dma_smartbond.h

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2023 Renesas Electronics Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef DMA_SMARTBOND_H_
+#define DMA_SMARTBOND_H_
+
+
+enum dma_smartbond_trig_mux {
```

Comment:
Since this is a public header please add some doxygen comments

Reply:
Done